src/features/expenses-list/expenses-list.routes-mock.ts

```diff
@@ -0,0 +1,71 @@
+import { Router } from "express";
+import { promises as fs } from "fs";
+import path from "path";
+export const router = Router();
+
+router.get("/", async (req, res) => {
```

Comment:
use zod's valudate

Reply:
the file not necessery anymore, maybe will add again in the test branch